Source/WebCore/accessibility/isolatedtree/AXIsolatedObject.cpp

```diff
@@ -2337,19 +2339,19 @@ AXIsolatedObject* AXIsolatedObject::headerContainer()
 
 unsigned AXIsolatedObject::headingTagLevel() const
 {
-    TagName tag = propertyValue<TagName>(AXProperty::TagName);
-    switch (tag) {
-    case TagName::h1:
+    ElementName name = propertyValue<ElementName>(AXProperty::ElementName);
```

Comment:
`auto` seems appropriate here since we specify the type of `ElementName` already as a template parameter to `propertyValue`

Also, any reason we don't just call the `elementName()` function you added?